gemstash.gemspec

```diff
@@ -28,7 +28,12 @@ you push your own private gems as well."
   spec.executables   = spec.files.grep(%r{^exe/}) {|f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "activesupport", "~> 4.2"
+  if RUBY_VERSION > "2.3"
```

Comment:
We can try it out and hope!

Those who want to keep `activesupport` back on their installations can use a Gemfile should they need to.